sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -2147,7 +2150,14 @@
                       "spellcheck": true,
                       "truncateMultipleSpaces": false,
                       "id": "ejhdok9",
-                      "tags": []
+                      "tags": [],
+                      "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": ""
+                      },
+                      "customConditional": "show = data.myProgramNotListed ? data.myProgramNotListed.programnotListed : false;",
```

Comment:
Should the `data.myProgramNotListed.programnotListed` be converted to bool as 
```js
show = data.myProgramNotListed ? !!data.myProgramNotListed.programnotListed : false;
```

Reply:
data.myProgramNotListed.programnotListed is a boolean itself. Still do we require a conversion?